pkg/preflight/runtime/docker.go

```diff
@@ -0,0 +1,31 @@
+package runtime
+
+import (
+	"github.com/pkg/errors"
+	"os/exec"
+)
+
+// Preflight runs checks to make sure that the container runtime
+// is working as expected
+func Preflight() error {
+	checks := []func() error{
+		dockerIsRunning,
+	}
+
+	for _, check := range checks {
+		if err := check(); err != nil {
+			return errors.Wrap(err, "Preflight check failed")
```

Comment:
error strings should start with lowercase.
[1]

Reply:
Thanks, I've fixed it. :)